app/graph/mutations/media_mutations.rb

```diff
@@ -4,8 +4,7 @@ module MediaMutations
     account_id: 'int',
     project_id: 'int',
```

Comment:
Yes Caio and agreed with Karim to do this later, for now I created a ticket #5587